stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -367,17 +368,28 @@ public class MessageListViewModel @JvmOverloads constructor(
                 )
             }
             is Event.ShowMessage -> {
-                domain.loadMessageById(
-                    cid,
-                    event.messageId,
-                    MESSAGES_LIMIT,
-                    MESSAGES_LIMIT
-                ).enqueue { result ->
-                    if (result.isSuccess) {
-                        _targetMessage.value = result.data()
-                    } else {
-                        val error = result.error()
-                        logger.logE("Could not load message: ${error.message}. Cause: ${error.cause?.message}")
+                val message = messageListData?.value
```

Comment:
Search if the message is already available in the viewmodel does a great performance improvement. Without what, it is necessary to wait 1+ sec for the scroll actually happen